### PR TITLE
Fix backAndroid warning

### PR DESCRIPTION
**Pull request by GregoryNative**
Pull request created for fixing this warning:
"Warning: BackAndroid is deprecated.  Please use BackHandler instead."

### CODE REVIEW

File: src/PlatformHelpers.native.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+import {
+  BackAndroid as OldBackAndroid,
+  BackHandler as NewBackAndroid,
+  Linking,
+} from 'react-native';
 
-export { BackAndroid, Linking };
+const fullVersion = require('react-native/package.json').version.split('.')[1];
+
+let BackHandler = fullVersion < 44 ? OldBackAndroid : NewBackAndroid;
```
Comment (chirag04) [inline src/PlatformHelpers.native.js]: just look for BackHandler and use it if available otherwise use BackAndroid. no need to check for RN version

Comment (GregoryNative) [inline src/PlatformHelpers.native.js]: yeap, you are right.

File: src/PlatformHelpers.native.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
 /* @flow */
 
-import { BackAndroid, Linking } from 'react-native';
+import {
+  BackAndroid as OldBackAndroid,
+  BackHandler as NewBackAndroid,
```
Comment (skevy) [inline src/PlatformHelpers.native.js]: Import this just as BackHandler.

File: src/PlatformHelpers.native.js [hunk 3]
```diff
@@ -1,5 +1,11 @@
 /* @flow */
 
-import { BackAndroid, Linking } from 'react-native';
+import {
+  BackAndroid as OldBackAndroid,
```
Comment (skevy) [inline src/PlatformHelpers.native.js]: Call this `DeprecatedBackAndroid`.

**Pull request merged**